source/GameData.cpp

```diff
@@ -946,6 +959,18 @@ void GameData::LoadFile(const string &path, bool debugMode)
 			for(const DataNode &child : node)
 				list.push_back(child.Token(0));
 		}
+		else if(key == "categories" && node.Size() >= 2)
+		{
+			if(node.Token(1) == "outfit" || node.Token(1) == "ship")
+			{
+				vector<string> &categoryList = (node.Token(1) == "outfit" ? outfitCategories : shipCategories);
+				for(const DataNode &child : node)
+					if(find(categoryList.begin(), categoryList.end(), child.Token(0)) == categoryList.end())
+						categoryList.push_back(child.Token(0));
+			}
+			else
+				node.PrintTrace("Skipping unsupported use of categories object:");
```

Comment:
Personally I would quote the special token in the warning message, i.e.
`...use of \"categories\":"`